1-JavaScript/Week3/homework/ex5-shoppingCartPure.js

```diff
@@ -16,8 +16,13 @@ it pure. Do the following:
 5. Confirm that you function passes the provided unit tests.
 ------------------------------------------------------------------------------*/
 // ! Function under test
-function addToShoppingCart(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function addToShoppingCart(shoppingCart, item) {
+  const currentCart = [...shoppingCart];
```

Comment:
```suggestion
  const currentCart = shoppingCart;
```

shoppingCart is already an array, so you don't need to put it in a new array

Reply:
Thanks for the feedback Dayan, I changed that.